_episodes/02-sql-aggregation.md

```diff
@@ -6,7 +6,7 @@ questions:
 - "How can I summarize my data by aggregating, filtering, or ordering query results?"
 - "How can I make sure column names from my queries make sense and aren't too long?"
 objectives:
-- "Apply aggregation to group records in SQL."
+- "Apply aggregation to grouped records in SQL."
```

Comment:
It took me a couple tries to parse this sentence, but this is the verb "group", eg. "Group records together in SQL by applying aggregation". Maybe adding "together" somewhere will make this clearer :thinking: